azure/packages/test/scenario-runner/src/logger.ts

```diff
@@ -120,20 +121,24 @@ export async function getLogger(
 	config: LoggerConfig,
 	events?: string[],
 	transformEvents?: Map<string, string>,
-): Promise<TelemetryLogger> {
+): Promise<ITelemetryLogger> {
 	const baseLogger = await loggerP;
 	if (events) {
 		baseLogger.registerExpectedEvent(events);
 	}
 	if (transformEvents) {
 		baseLogger.transformEvents(transformEvents);
 	}
-	return ChildLogger.create(baseLogger, config.namespace, {
-		all: {
-			runId: config.runId,
-			scenarioName: config.scenarioName,
-			endpoint: config.endpoint,
-			region: config.region,
+	return createChildLogger({
+		logger: baseLogger,
```

Comment:
Subjective nit:  Maybe call the param property `baseLogger` instead of `logger`?

Reply:
it takes more bits, and i don't think it provides much more clarity, so i'd rather stick with the shorter name